portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -29,56 +29,73 @@ class ProjectControllerItTest
   override protected def beforeAll(): Unit = {
     super.beforeAll()
     components.datastorageModule.pipelineDatabaseEngine.updateSchema()
+    userRepository.addUser(stranger).flatMap { _ =>
+      userRepository.addUser(dummyUser).flatMap { _ =>
+        projectRepository.addProject(dummyProject).map(_ => ())
+      }
+    }
   }
 
   private val dummyUser = TestUserUtils.getDummyUser()
+  private val stranger = TestUserUtils.getDummyUser()
+  private val dummyProject = TestProjectUtils.getDummyProject(ownerId = dummyUser.userId)
 
   "ProjectController" when {
     "getProjectByName" should {
       "return a project with the same name" in {
-        val dummyProject =
-          TestProjectUtils.getDummyProject(ownerId = dummyUser.userId)
         val projectByNameRequest = dummyProject.name
+        val accessToken = AccessTokenContent(dummyProject.ownerId)
+
+        Get("/projects?name=" + projectByNameRequest) ~> projectController.route(accessToken) ~> check {
+          status shouldBe StatusCodes.OK
+          responseAs[Option[Project]] shouldEqual Option(dummyProject)
+        }
+      }
+
+      "return status code 403 if project was gotten wrong user" in {
```

Comment:
You still need to clarify what is wrong with this user :) We should be able to understand what is the test purpose by its name only.

Reply:
Ok, sorry. My fault. Fixed.